File: solutions/26/go/26.removeduplicatesfromsortedarray.go
func removeDuplicates(nums []int) int {
    numberLength := len(nums)
    if numberLength < 2{
        return numberLength
    }
    
    idx := numberLength - 2
    for idx >= 0 {
        if nums[idx] == nums[idx + 1] {
            nums = append(nums[:idx], nums[idx+1:]...)
        }
        idx--
    }
    return len(nums)
}

File: ssolutions/26/javascript/26.Remove Duplicates from Sorted Array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
const removeDuplicates = (nums) => {
    let numLength = nums.length;
    if (numLength < 2) {
        return numLength;
    }
    for (let i = numLength - 2; i >= 0; i--) {
        if (nums[i] === nums[i + 1]) {
            nums.splice(i, 1);
        }
    }
    return nums.length;
};